test/integration/scheduler/framework_test.go

```diff
@@ -748,7 +797,7 @@ func TestScorePlugin(t *testing.T) {
 				}
 			}
 
-			if scorePlugin.numScoreCalled == 0 {
+			if numScoreCalled := atomic.LoadInt32(&scorePlugin.numScoreCalled); numScoreCalled == 0 {
```

Comment:
as mentioned in https://github.com/kubernetes/kubernetes/pull/93534#pullrequestreview-460348120
> BTW: can you also update L792 to make the scoreNum is obtained atomically?

Sure, I updated to read `numScoreCalled` with `atomic` in `TestPostFilterPlugin`, too.